contracts/minters/vending-minter/src/contract.rs

```diff
@@ -243,9 +245,37 @@ pub fn execute(
             execute_update_discount_price(deps, env, info, price)
         }
         ExecuteMsg::RemoveDiscountPrice {} => execute_remove_discount_price(deps, info),
+        ExecuteMsg::ReceiveNft(msg) => execute_burn_to_mint(deps, env, info, msg),
     }
 }
 
+pub fn execute_burn_to_mint(
```

Comment:
Duplicate code. Import from another contract or package.

Reply:
Resolved per this can not be put in another package.